components/NavDrawer/index.jsx

```diff
@@ -243,33 +245,56 @@ class NavDrawer extends Component {
         {showQuickLinks && this.renderEntries(entries.quicklinks)}
         {showQuickLinks && <Divider close={this.handleClose} />}
 
-        {/* Shipping */}
-        <Portal name={marketPortals.NAV_MENU_SHIPPING_BEFORE} props={props} />
```

Comment:
Those need to move to more menu. See PWA-715

Reply:
No, this was desired behavior.